### PR TITLE
feat(view): add translate

### PR DESCRIPTION
**Pull request by lrangelc**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (ruedagato) [inline .vscode/launch.json:1]: skip this file for the repo

File: apps/guest/src/assets/i18n/en-US.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "form": {
+      "fields": {
+        "user": {
+          "label": "User",
+          "placeholder": "test@test.com",
+          "message-error": "We need an email address to log in"
+        },
+        "password": {
+          "label": "Password",
+          "placeholder": "********",
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:21]: if there is no difference between language, not is necessary to add this for translate

File: apps/guest/src/assets/i18n/en-US.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          "message-error": "We need a password to log in"
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:22]:
```suggestion
          "messageError": "We need a password to log in"
```

File: apps/guest/src/assets/i18n/en-US.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "youDoNotHavePermissionToAccess": "You do not have permission to access",
+    "yourEntryDateIs": "Your entry date is",
+    "weInviteYouToLogInOnThisDate": "We invite you to log in on this date",
+    "yourDepartureDateWas": "Your departure date was",
+    "form": {
+      "fields": {
+        "user": {
+          "label": "User",
+          "placeholder": "test@test.com",
+          "message-error": "We need an email address to log in"
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:17]:
```suggestion
          "messageError": "We need an email address to log in"
```

File: apps/guest/src/assets/i18n/en-US.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+  "auth/user-not-found": "Invalid email or password",
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:2]:
```suggestion
  "auth": {
      "userNotFound": "Invalid email or password",
      ...
  }
```

File: apps/guest/src/assets/i18n/en-US.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  "auth/wrong-password": "Invalid email or password",
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:3]: same above

File: apps/guest/src/assets/i18n/en-US.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  "auth/user-disabled": "The account is no longer active",
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:4]: same above

File: apps/guest/src/assets/i18n/en-US.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+  "login": {
+    "title": "Enter with the username and password that will be provided to you when you check in at the hotel",
+    "didYouForgetYourPassword": "Did you forget your password?",
+    "youDoNotHavePermissionToAccess": "You do not have permission to access",
+    "yourEntryDateIs": "Your entry date is",
+    "weInviteYouToLogInOnThisDate": "We invite you to log in on this date",
+    "yourDepartureDateWas": "Your departure date was",
+    "form": {
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:12]: it isn't necessary this level of specificity, with `fields user` and` fields password ` inside login is quite clear

File: apps/guest/src/assets/i18n/en-US.json [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+          "placeholder": "test@test.com",
+          "message-error": "We need an email address to log in"
+        },
+        "password": {
+          "label": "Password",
+          "placeholder": "********",
+          "message-error": "We need a password to log in"
+        }
+      },
+      "actions": { "submit": "ENTER" }
```
Comment (ruedagato) [inline apps/guest/src/assets/i18n/en-US.json:25]:
```suggestion
      "actionsSubmit": "ENTER" 
```

**Review by ruedagato — CHANGES_REQUESTED** [review]

File: apps/guest/src/assets/i18n/en-US.json (hunk 9 above)
Comment (ruedagato) [reply apps/guest/src/assets/i18n/en-US.json:25]: intente usar hasta dos niveles de profundidad para el lenguaje de archivos JSON

File: .vscode/launch.json (hunk 1 above)
Comment (lrangelc) [reply .vscode/launch.json:1]: into .gitignore it's
# IDE - VSCode
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json

File: apps/guest/src/assets/i18n/en-US.json (hunk 5 above)
Comment (lrangelc) [reply apps/guest/src/assets/i18n/en-US.json:2]: I am trying to use the error code directly and not make any other changes

File: .vscode/launch.json (hunk 1 above)
Comment (ruedagato) [reply .vscode/launch.json:1]: ok, I will delete it after

**Review by ruedagato — APPROVED** [review]

**Pull request merged**